packages/snap/src/index.ts

```diff
@@ -32,92 +38,100 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
   }
 
   const { tx } = request.params;
-  tx.value = handleNumber(tx.value); // todo find way to do this within io-ts transformation
+  tx.value = handleNumber(tx.value);
   const network = await getNetworkConfig();
   const provider = new CeloProvider(network.url);
   const keyPair = await getKeyPair(snap, tx.from);
   const wallet = new CeloWallet(keyPair.privateKey).connect(provider);
+  tx.from = tx.from ? tx.from : wallet.address;
```

Comment:
not sure why this line was added. 

If `tx.from` does not exist, than `wallet` cannot be generated, since `keypair` is using `tx.from`

If `tx.from` could in some cases be missing, should there be a check before line 44 that `tx.from` actually exist?